src/datasource/render/frontend.ts

```
@@ -518,6 +552,21 @@ export function getOwnerInfo(
   );
 }
 
+export function getQueryParameterInfo(
+  chunkManager: ChunkManager,
+  hostname: string,
+  options: Partial<ProgressOptions>,
+): Promise<QueryParameterInfo[]> {
+  return chunkManager.memoize.getAsync(
+    { type: "render:getQueryParameterInfo", hostname },
+    options,
+    (progressOptions) =>
+      fetchOk(`${hostname}/render-ws/swagger.json`, progressOptions)
```

Possibly add a `.catch()` to this function, with a more useful error message.

---

The output of this is shown as part of a UI element that does its own error handling, so I think I should propagate any errors upward. Also, `parseQueryParameterInfo` will always return a default result, even if the fetch failed. I added logging statements for that case, though.